<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPanel</class>
 <widget class="QWidget" name="ControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>889</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Панель управления умным домом</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #333333;  /* Тёмно-серый */</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="styleSheet">
      <string notr="true">font: bold 16pt;</string>
     </property>
     <property name="text">
      <string>Управление устройствами</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft">
    <widget class="QTimeEdit" name="timeEdit">
     <property name="displayFormat">
      <string>HH:mm</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupKettle">
     <property name="title">
      <string>Чайник</string>
     </property>
     <layout class="QGridLayout" name="gridKettle">
      <item row="2" column="2">
       <widget class="QLabel" name="displayKettleSetTemp">
        <property name="text">
         <string>80°C</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSlider" name="sliderKettleTemp">
        <property name="minimum">
         <number>40</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>80</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btnKettleOn">
        <property name="text">
         <string>Включить сейчас</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnKettleSchedule">
        <property name="text">
         <string>Запланировать</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="currentKettleTemp">
        <property name="text">
         <string>-- °C</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelKettleTemp">
        <property name="text">
         <string>Температура воды:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTimeEdit" name="timeKettle">
        <property name="displayFormat">
         <string>HH:mm</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelSetKettleTemp">
        <property name="text">
         <string>Установить температуру:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupAC">
     <property name="title">
      <string>Кондиционер</string>
     </property>
     <layout class="QGridLayout" name="gridAC">
      <item row="0" column="0">
       <widget class="QLabel" name="labelACTemp">
        <property name="text">
         <string>Температура в комнате:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="currentACTemp">
        <property name="text">
         <string>-- °C</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboACMode">
        <item>
         <property name="text">
          <string>Охлаждение</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Обогрев</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Вентиляция</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Авто</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnACSwitch">
        <property name="text">
         <string>Вкл/Выкл</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelACSetTemp">
        <property name="text">
         <string>Установить темп.:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSlider" name="sliderACSetTemp">
        <property name="minimum">
         <number>16</number>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
        <property name="value">
         <number>22</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="displayACSetTemp">
        <property name="text">
         <string>22°C</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btnClimateControl">
        <property name="text">
         <string>Включить климат-контроль</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupLight">
     <property name="title">
      <string>Свет</string>
     </property>
     <layout class="QGridLayout" name="gridLight">
      <item row="0" column="0">
       <widget class="QPushButton" name="btnLightSwitch">
        <property name="text">
         <string>Вкл/Выкл</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTimeEdit" name="timeLight">
        <property name="displayFormat">
         <string>HH:mm</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnLightSchedule">
        <property name="text">
         <string>Запланировать</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupSocket">
     <property name="title">
      <string>Умная розетка</string>
     </property>
     <layout class="QHBoxLayout" name="hboxSocket">
      <item>
       <widget class="QPushButton" name="btnSocketSwitch">
        <property name="text">
         <string>Вкл/Выкл</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeSocket">
        <property name="displayFormat">
         <string>HH:mm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSocketSchedule">
        <property name="text">
         <string>Запланировать</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupCurtain">
     <property name="title">
      <string>Шторы</string>
     </property>
     <layout class="QHBoxLayout" name="hboxCurtain">
      <item>
       <widget class="QPushButton" name="btnCurtainOpen">
        <property name="text">
         <string>Открыть</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCurtainClose">
        <property name="text">
         <string>Закрыть</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeCurtain">
        <property name="displayFormat">
         <string>HH:mm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCurtainSchedule">
        <property name="text">
         <string>Запланировать</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="musicGroupBox">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>200</height>
        </size>
       </property>
       <property name="title">
        <string>Музыка</string>
       </property>
       <widget class="QProgressBar" name="progressBar">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>291</width>
          <height>28</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>260</width>
          <height>20</height>
         </size>
        </property>
        <property name="value">
         <number>50</number>
        </property>
       </widget>
       <widget class="QComboBox" name="playlistComboBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>260</width>
          <height>28</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>260</width>
          <height>25</height>
         </size>
        </property>
        <property name="currentText">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="songLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>260</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>260</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Текущий трек</string>
        </property>
       </widget>
       <widget class="QSlider" name="volumeSlider">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>100</y>
          <width>261</width>
          <height>51</height>
         </rect>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="volumeLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>108</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Громкость: 50%</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>256</width>
          <height>32</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="prevButton">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>⏮</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="playButton">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string> ▶</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="nextButton">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>⏭</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
